common/src/main/java/com/wynntils/features/user/DummyOverlayFeature.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.annotations.OverlayHolder;
+import com.wynntils.core.features.overlays.annotations.RegisteredOverlay;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.utils.objects.CustomColor;
+
+@OverlayHolder
+public class DummyOverlayFeature extends UserFeature {
+    @RegisteredOverlay
```

Comment:
I'll assume you want to do more with this later, otherwise this is also useless if you use the sorting method I mentioned above

Reply:
I mean, it is already used for stuff like render hooking and render "priority", although I should make it not limited to Pre and Post, but rather make it have a priority int.